### PR TITLE
Disable shard replication by default

### LINKED ISSUE #6972: Update Stackgres Deployment To Not Use Replicas
### Problem

We no longer want to deploy our citus cluster with read replicas. 

### Solution

Update the configuration to have only one instance per shard.

### Alternatives

_No response_

### PR DESCRIPTION
**Pull request by jnels124**
**Description**:
disable replicas for workers

**Related issue(s)**:

Fixes #6972 

**Notes for reviewer**:
Only disabled replicas for workers.

**Checklist**

- [ ] Documented (Code comments, README, etc.)
- [ ] Tested (unit, integration, etc.)

### CODE REVIEW

File: charts/hedera-mirror/values-prod.yaml [hunk 1]
```diff
@@ -121,7 +121,6 @@ stackgres:
       storageClass: zfs
   worker:
     instances: 3
-    replicasPerInstance: 2
```
Comment (steven-sheehy) [inline charts/hedera-mirror/values-prod.yaml:124]: Prefer we keep the property but explicitly set it to 1.

Comment (jnels124) [reply charts/hedera-mirror/values-prod.yaml:124]: got it. Removed it from here as we specify the default in values.yaml. Not against having it here to though.

**Review by xin-hedera — APPROVED** [review]
LGTM

**Review by steven-sheehy — APPROVED** [review]
LGTM

**Comment by steven-sheehy** [comment]
Has this change been applied to the cluster yet?

**Pull request merged**